### ISSUE TITLE
Minicart count not in sync

### ISSUE THREAD

**Issue by naltatis** [heerens/tractor-store-htmx-tailwind#1]
I noticed that the minicart shows an outdated count when navigating to a previously visited site. Is this a caching issue?

[minicart.webm](https://github.com/user-attachments/assets/e1a22c45-a37a-4aad-83f1-887a25869be3)

**Comment by heerens** [comment]
Good catch, thanks! Yes its a caching config issue. I'll have a look...

**Comment by heerens** [comment]
Fixed in https://github.com/heerens/tractor-store-htmx-tailwind/commit/09395c2837551203d3f2fca8f2d92c0aba5c6f7d
